lib/components/navbar/NavLink.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from 'next/link';
+import { withRouter } from 'next/router';
+
+const NavLink = (props: any) => (
+  <Link href={props.path} passHref>
+    <div className={`NavButton ${props.router.pathname === props.path ? 'active' : ''}`}>
+      <span className="Label">{props.label}</span>
+    </div>
+  </Link>
+);
+
+export default withRouter(NavLink);
```

Comment:
🤔 Why not use the `useRouter` hook? https://nextjs.org/docs/api-reference/next/router